tests/test-sinkinterface.c

```diff
@@ -7,58 +7,60 @@
 #include "src/ngf/request-internal.h"
 #include "src/ngf/core-player.c"
 
-
 START_TEST (test_get_core_and_name)
 {
-    NSinkInterface *iface = NULL;
-    iface = g_new0 (NSinkInterface, 1);
-    ck_assert (iface != NULL);
-    
-    ck_assert (n_sink_interface_get_core (NULL) == NULL);
-    ck_assert (n_sink_interface_get_name (NULL) == NULL);
+    static const char fake_sink_name[] = "TEST_GET_CORE_AND_NAME_sink_name";
+    static const char fake_path[] = "TEST_GET_CORE_AND_NAME_path";
 
-    NCore *core = NULL;
-    core = n_core_new (NULL, NULL);
+    NCore *core = n_core_new (NULL, NULL);
     ck_assert (core != NULL);
-    core->conf_path = g_strdup ("path");
+    core->conf_path = g_strdup (fake_path);
+
+    NSinkInterface *iface = g_new0 (NSinkInterface, 1);
+    ck_assert (iface != NULL);
```

Comment:
For minor detail, I suppose these shouldn't have much chance of failing. And on a similar vein, the function cleanup parts assigning stuff to null as last things seems a bit pointless when there's clearly nothing that could mess up with them. Up to you if you feel like removing these.

Reply:
Yup. Removed alloc asserts.